logicrunner/builtin/contract/member/member.go

```diff
@@ -211,14 +243,14 @@ func (m *Member) addBurnAddressesCall(params map[string]interface{}) (interface{
 		return nil, fmt.Errorf("only migration daemon admin can call this method")
 	}
 
-	burnAddressesStr := make([]string, len(burnAddressesI))
-	for i, ba := range burnAddressesI {
-		burnAddressesStr[i] = ba.(string)
-	}
+	migrationAddressesStr := make([]string, len(migrationAddresses))
 
-	err := rootDomain.AddMigrationAddresses(burnAddressesStr)
+	for i, ba := range migrationAddresses {
+		migrationAddressesStr[i] = ba.(string)
```

Comment:
add check that it can be casted to string

Reply:
done